app/templates/events/view/tickets/attendees.hbs

```diff
@@ -0,0 +1,76 @@
+<div class="ui hidden divider"></div>
+<div class="ui compact menu pull right floated">
+  <div class="ui simple dropdown item">
+    {{t 'Showing all records'}}
+    <i class="dropdown icon"></i>
+    <div class="menu">
+      <div class="item"><i class="calendar icon"></i>{{t 'Filter by Date'}}</div>
+      <div class="item"><i class="ticket icon"></i>{{t 'Filter by Ticket'}}</div>
+      <div class="item"><i class="remove circle icon"></i>{{t 'Remove all Filters'}}</div>
+    </div>
+  </div>
+</div>
+<div class="ui compact menu pull right floated">
+  <div class="ui simple dropdown item">
+    {{t 'Export As'}}
+    <i class="dropdown icon"></i>
+    <div class="menu">
+      <div class="item"><i class="file pdf outline icon"></i>{{t 'PDF'}}</div>
+      <div class="item"><i class="file excel outline icon"></i>{{t 'CSV'}}</div>
+    </div>
+  </div>
+</div>
+<div class="content">
+  <h2 class="ui header">{{t 'View Attendees'}}</h2>
+</div>
+<div class="ui hidden divider"></div>
+  <div class="ui stackable basic buttons">
+    <div class="ui button active">{{t 'Completed'}}</div>
+    <div class="ui button">{{t 'Placed'}}</div>
+    <div class="ui button">{{t 'Pending'}}</div>
+    <div class="ui button">{{t 'Expired'}}</div>
+    <div class="ui button">{{t 'Cancelled'}}</div>
+    <div class="ui button">{{t 'Checked In'}}</div>
+    <div class="ui button">{{t 'Not Checked In'}}</div>
+    <div class="ui button">{{t 'All'}}</div>
+  </div>
+<table class="ui celled table">
+  <thead>
+    <tr>
+      <th>#</th>
+      <th>{{t 'Order'}}</th>
+      <th>{{t 'Ticket Name'}}</th>
+      <th>{{t 'Ticket Price'}} (US$)</th>
+      <th>{{t 'First Name'}}</th>
+      <th>{{t 'Last Name'}}</th>
+      <th>{{t 'Email'}}</th>
+      <th>{{t 'Action'}}</th>
+    </tr>
+  </thead>
+  <tbody>
+    {{#each model as |attendees|}}
+      <tr>
+        <td>{{attendees.id}}</td>
+        <td>{{attendees.invoice_number}}</td>
+        <td>{{attendees.ticket_name}}</td>
+        <td>US $ {{attendees.ticket_price}}</td>
+        <td>{{attendees.first_name}}</td>
+        <td>{{attendees.last_name}}</td>
+        <td>{{attendees.email}}</td>
+        <td class="centre aligned">
+          <div class="ui vertical compact basic buttons">
```

Comment:
This makes no sense. This is attendees route, not orders. These are the wrong actions. These are the actions required for attendees route.  Please check this.
![image](https://cloud.githubusercontent.com/assets/17252805/26773354/a136af08-49e8-11e7-8bbb-05ddb874b93d.png)